### ISSUE TITLE
Tooltip is not working with 'ember-infinity'

### ISSUE THREAD

**Issue by jigneshchudasama05** [sir-dunxalot/ember-tooltips#45]
Tooltip content is not showing on infinite scrolled content.
it's working on first-page content but when infinite scrolling content comes it's not working on it.

View
```
import Ember from 'ember';
import TooltipsMixin from 'ember-tooltips/mixins/components/tooltips';

export default Ember.Component.extend(TooltipsMixin, {
    didInsertElement: function() {
        this.renderChildTooltips();
    }
});

```
and tooltip content structure,

```<div class="has-tooltip" data-tooltip-content='test'>Test</div>```

**Comment by sir-dunxalot** [comment]
This might be to do with the way the tooltip elements are added as children of the body tag, not the rendering element they appear attached to. The data attributes approach is also currently sub par, as is evident by #5 and #43.

Are you able to test this using a component and `tooltipContent` property instead of the `data-<x>` attributes? Or do you have a dummy app you can share that replicates this?

**Comment by jigneshchudasama05** [comment]
You have any idea how to call view from route?

```

Router:

import Ember from 'ember';
import InfinityRoute from 'ember-infinity/mixins/route';

export default Ember.Route.extend(InfinityRoute, {
    model: function(params) {
        return this.infinityModel('log', Ember.$.extend({ 
            perPage: 20,
        }, params));
    },
    infinityModelUpdated: function() {
        // Call view from here 
    }
});

```

```
View: 

import Ember from 'ember';
import TooltipsMixin from 'ember-tooltips/mixins/components/tooltips';

export default Ember.View.extend(TooltipsMixin, {
    initializePlugins: function () {
        this.renderChildTooltips();
    }.on('didInsertElement'),
});
```

**Comment by sir-dunxalot** [comment]
If you had to do it that way, you could send an action from the view route, with the views context as a param. That way, the view could 'register' itself with the route on init. Seems a little backwards though.

**Comment by sir-dunxalot** [comment]
I am hoping this issue is now fixed with release `1.0.0`: https://github.com/sir-dunxalot/ember-tooltips/releases/tag/1.0.0

If that is incorrect we can reopen this issue.
